python/nano/src/bigdl/nano/deps/openvino/core/model.py

```diff
@@ -153,3 +172,42 @@ def pot(self,
 
     def _model_exists_or_err(self):
         invalidInputError(self.ie_network is not None, "self.ie_network shouldn't be None.")
+
+    def async_predict(self,
+                      input_data: Union[List[np.ndarray], List[List[np.ndarray]]],
+                      num_requests: int = 0) -> List[np.ndarray]:
+        """
+        Perfrom model inference using async mode.
+
+        :param input_data: Input data to be inferenced.
+                           Users can put multiple input data in a list to infer them together.
+                           Can be List[numpy.ndarray] or
+                           List[List[numpy.ndarray]] if the model has multiple inputs.
+        :param num_requests: Number of requests in the asynchronous infer requests pool.
+                             Each element in input_data will be bound to an idle async
+                             infer request in the pool to do inference.
+                             Defaults to 0.
+                             If 0, it will be set automatically to the optimal number.
```

Comment:
I understand what will happen if num_requests is set > 1. Could you have some test on the compatibility between nano's env variables(open by `source bigdl-nano-init`) and this feature?

Reply:
I am worrying about the core binding of intel openmp might affect this function

Reply:
> I understand what will happen if num_requests is set > 1. Could you have some test on the compatibility between nano's env variables(open by `source bigdl-nano-init`) and this feature?

You mean run test of async api when the nano's env variables are set by `source bigdl-nano-init`?